lib/gollum/public/gollum/javascript/HOWTO_UPDATE_ACE.md

```diff
@@ -1,8 +1,8 @@
-Current ACE version: 1.4.4
-
 # How to update the ACE editor component
 
 1. Download a new ACE release from (ajax.org/ace-builds)[https://github.com/ajaxorg/ace-builds/releases].
 2. Extract the `src-min-noconflict` folder from the archive and rename it to `ace-x.y.z`, where `x.y.z` matches exactly the release version.
-3. Replace the version in `app.js` to `x.y.z`.
-4. Test if the editor works.
+3. Place the version in the javascript directory
+4. Update the symlink 'ace' to point to the new version
```

Comment:
Will symlinks work on Windows?

Reply:
Also note: this is only an issue for developing/running from source. When running with precompiled assets the symlink is not an issue at all.